godot-macros/src/godot_api.rs

```diff
@@ -5,8 +5,8 @@
  */
 
 use crate::method_registration::{make_method_registration, make_virtual_method_callback};
-use crate::util;
 use crate::util::bail;
+use crate::util::{self, KvParser};
```

Comment:
Avoid `self` in imports, in accordance with [code style](https://godot-rust.github.io/book/gdext/contribute/conventions.html#code-organization) guidelines.